.github/workflows/link-checker.yaml

```
@@ -0,0 +1,38 @@
+---
+name: Link Checker
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  link-checker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Link Checker
+        id: lychee
+        uses: lycheeverse/lychee-action@v1.3.2
+        env:
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+        with:
+          args:  --exclude-all-private './content/**/*.md' -u 'Mozilla/5.0 (X11; Linux x86_64; rv:10.0) Gecko/20100101 Firefox/10.0' --exclude http:* mongodb:*
```

Excluding all of `http:` feels a little too broad.  Is it possible to be a little more specific?  `--exclude-all-private` should already cover the localhost cases, and we could explicitly ignore any sample-project links... what other cases need to be excluded?

```suggestion
          args:  --exclude-all-private './content/**/*.md' -u 'Mozilla/5.0 (X11; Linux x86_64; rv:10.0) Gecko/20100101 Firefox/10.0' --exclude http://sample-project-* mongodb:*
```